src/Build/BackEnd/Shared/EventsCreatorHelper.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Shared;
+
+namespace Microsoft.Build.BackEnd.Shared;
+
+internal static class EventsCreatorHelper
+{
+    public static BuildMessageEventArgs CreateMessageEventFromText(BuildEventContext buildEventContext, MessageImportance importance, string message, params object?[]? messageArgs)
+    {
+        ErrorUtilities.VerifyThrow(buildEventContext != null, "buildEventContext was null");
+        ErrorUtilities.VerifyThrow(message != null, "message was null");
```

Comment:
added parameterName too:
```csharp
ErrorUtilities.VerifyThrowArgumentNull(buildEventContext, nameof(buildEventContext));
ErrorUtilities.VerifyThrowArgumentNull(message, nameof(message));
```

Reply:
it should be `VerifyThrowInternalNull` instead of `VerifyThrowArgumentNull` for the same reason https://github.com/dotnet/msbuild/pull/10272/files#diff-d1e67edbe71d7c812b189c453fa53ec5c9057aa29c2b0877a482dec3b135ea10L29